Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyFragment.kt

```diff
@@ -28,6 +28,9 @@ import androidx.fragment.app.viewModels
 import com.example.compose.jetsurvey.R
 import com.example.compose.jetsurvey.theme.JetsurveyTheme
 import com.google.android.material.datepicker.MaterialDatePicker
+import java.text.SimpleDateFormat
+import java.util.Date
```

Comment:
these imports are no longer needed.
make sure you run `spottlessApply`

Reply:
fixed